function/fetchFile.js

```diff
@@ -0,0 +1,61 @@
+/* Form of a Google Cloud Function URL (https://cloud.google.com/functions/docs/calling/http)
+   "https://YOUR_REGION-YOUR_PROJECT_ID.cloudfunctions.net/FUNCTION_NAME"
+*/
+
+//HTML that saves the breed and score from the returned JSON body
+<div id="prediction-results" style="display: none">
+<span class="label">Breed: </span><span id="dog-breed" class="score"></span> //Can access with getElementById
+<span class="label">Confidence: </span><span id="prediction-score" class="score"></span> //Can access with getElementById
+</div>
+
+//HTML for the error message
+<div id="prediction-error" style="display: none">
+<span class="p">ErrorMessage: </span><span id="error-message" class="score"></span> //Can access with getElementById
+</div>
+
+
+//Javascript
+var url = 'https://us-central1-sp19-codeu-35-7727.cloudfunctions.net/predict';
```

Comment:
@michelaburns You can test with https://us-central1-sp19-codeu-35-7727.cloudfunctions.net/testPredict in the meantime - it just returns a static object with `label` and `score` keys. Use this to make sure your JavaScript successfully gets a result + populates the HTML elements. We can merge + deploy that, and then swap to our "production" function once everything looks OK.

Code is at: https://console.cloud.google.com/functions/details/us-central1/testPredict?project=sp19-codeu-35-7727&tab=source

Reply:
Okay! Just to confirm, I should replace the code in testPredict with my javascript?

Reply:
No - just replace the URL here.

e.g. integrate against testPredict instead of predict for now.

Reply:
Okay gotcha haha. I'll change that in the code right now!